apps/studio/components/interfaces/Integrations/CronJobs/CronJobScheduleSection.tsx

```diff
@@ -1,107 +1,221 @@
+                    </>
+                  </FormControl_Shadcn_>
+
+                  <div className="flex items-center gap-2 mt-2">
+                    <Switch
+                      checked={useNaturalLanguage}
+                      onCheckedChange={() => {
+                        setUseNaturalLanguage(!useNaturalLanguage)
+                        setInputValue('')
+                        setPresetValue('')
+                        setScheduleString('')
+                      }}
+                    />
+                    <p className="text-sm text-foreground-light">Use natural language</p>
+                  </div>
+
+                  <div className="mt-4">
+                    <p className="text-sm text-foreground-light">Use a preset</p>
+                    <ul className="flex gap-2 flex-wrap mt-2">
+                      {presets.map((preset) => (
+                        <li key={preset.name}>
+                          <Button
+                            type="outline"
+                            onClick={() => {
+                              setPresetValue(preset.expression)
+                              setInputValue(preset.expression)
+                              setUseNaturalLanguage(false)
+                            }}
+                          >
+                            {preset.name}
+                          </Button>
+                        </li>
+                      ))}
+                    </ul>
+                  </div>
+                  <Accordion_Shadcn_ type="single" collapsible className="mt-2">
+                    <AccordionItem_Shadcn_ value="item-1" className="border-none">
+                      <AccordionTrigger_Shadcn_ className="text-xs text-foreground-light font-normal gap-2 justify-start py-1 ">
+                        View syntax chart
+                      </AccordionTrigger_Shadcn_>
+                      <AccordionContent_Shadcn_>
+                        <CronSyntaxChart />
+                      </AccordionContent_Shadcn_>
+                    </AccordionItem_Shadcn_>
+                  </Accordion_Shadcn_>
                 </div>
               </FormControl_Shadcn_>
-              <FormMessage_Shadcn_ className="mt-1">
-                {scheduleString ? (
-                  <span className="text-foreground-light">
-                    The cron will be run {/* lowercase the first letter */}
-                    {scheduleString
-                      .split(' ')
-                      .map((s, i) => (i === 0 ? s.toLocaleLowerCase() : s))
-                      .join(' ')}
-                    .
-                  </span>
-                ) : (
-                  !fieldState.error && <span className="h-5" />
-                )}
+
+              <FormMessage_Shadcn_ className="mt-4">
+                <div className="mt-2 bg-surface-100 p-4 rounded grid gap-4 border">
+                  <h4 className="text-sm text-foreground">Schedule</h4>
+                  {scheduleString ? (
```

Comment:
this feels like repeated information with the input field imo
![image](https://github.com/user-attachments/assets/41992d73-2df2-4188-89fc-34988e0c7c25)

Reply:
im wondering if maybe the description here is all we need "the cron will be run at ___"

Reply:
hmm a bit on the fence for either way actually - will relook at this again

but i reckon RE the syntax chart, we could potentially line it up with the cron syntax in the schedule box - will be even clearly perhaps (and also one less repetition of * * * * * *
![image](https://github.com/user-attachments/assets/fa16a409-4ada-45df-acd9-e6d1fe9b67f4)

Reply:
Yeah it's meant to be a kind of "here's the summary of what's happening" 
it's redundant here in this view, but not when in natural language view. 

https://share.cleanshot.com/hJlwj6CX

Reply:
(that's the current reasoning, always open to suggestion tho!)